src/components/StyledListItem.js

```diff
@@ -0,0 +1,48 @@
+import {
+  CategoryAndDate,
+  LeftBlock,
+  RightBlock,
+  StyledAmount,
+  StyledCategory,
+  StyledDate,
+  StyledLink,
+} from "@/src/components/styles/List";
+import { CategoryIcon } from "@/src/components/CategoryIcon";
+
+// Utility function to get currency symbol
+const getCurrencySymbol = (currencyCode) => {
```

Comment:
not our "usual" way to write functions with =>

Reply:
Thanks for the hint! I'll change it accordingly :)